src/components/RightSidebar.tsx

```diff
@@ -48,32 +48,36 @@ export function RightSidebar({ content }: RightSidebarProps) {
             animate={{ x: 0, opacity: 1 }}
             transition={{ delay: 0.2, ease: [0.51, 0.92, 0.24, 1], duration: 0.3 }}
             exit={{ transition: { ease: "linear", duration: 0.2 }, x: "100%" }}
-            css={Css.w100.mw(RIGHT_SIDEBAR_MIN_WIDTH).z0.$}
+            css={Css.w100.mw(RIGHT_SIDEBAR_MIN_WIDTH).z0.maxh("calc(100vh - 120px)").oya.pl4.pr3.$}
           >
-            <div css={Css.relative.z0.px3.$}>
-              {/* Close button */}
-              <div css={Css.absolute.leftPx(-24).top0.$}>
-                <IconButton
-                  bgColor={Palette.White}
-                  circle
-                  onClick={() => setSelectedIcon(undefined)}
-                  icon="x"
-                  inc={3.5}
-                />
-                {/* vertical line */}
-                <div css={Css.absolute.topPx(48).leftPx(23).h("calc(100vh - 168px)").wPx(1).bgGray300.$} />
-              </div>
-
-              {/* Horizontal icons when opened */}
-              <div css={Css.df.aic.jcfe.gap2.mb3.$}>
-                <IconButtonList content={content} selectedIcon={selectedIcon} onIconClick={setSelectedIcon} />
+            <>
+              {/* Sticky header section */}
+              <div css={Css.sticky.top0.bgWhite.z999.$}>
```

Comment:
I had some difficulty getting all the `icons` to align horizontally and make only the left one sticky, so I ended up just putting them in the same `div` and making them both sticky. I think it should be ok, plus it looks nicer visually to see both stick